startup_assistance/indico_startup_assistance/controllers.py

```diff
@@ -0,0 +1,22 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
```

Comment:
let's use 2019 in newly added files

Reply:
Ok, I thought that we said we should keep it 2018 and just before release we will change all places in one go.

Reply:
yep we will update all the existing files in one go, but nothing wrong with updating the file template in your editor to use 2019 for new files :)